docs/reference/uds-core/idam/customization.md

```diff
@@ -281,3 +281,210 @@ The `SSO_SESSION_IDLE_TIMEOUT` specifies how long a session remains active witho
 
 The `SSO_SESSION_MAX_PER_USER` provides a limit on the number of active sessions a user can use. You can specify 0 to allow unlimited sessions per user, or set a specific number to limit concurrent sessions. This is useful for controlling resource usage and ensuring that users do not have an excessive number of active sessions at once.
+
+### OpenTofu Keycloak Client Configuration
+
+The UDS Identity Config includes a Keycloak client that can be used by OpenTofu to manage Keycloak resources programmatically. This client is disabled by default for security reasons.
+
+:::caution
+**Critical Security Requirements**
+
+1. **Pre-Deployment Configuration**
+   - **You must configure authentication flows before deploying UDS Core**
+   - UDS Core will apply default authentication flows if not configured first
+   - This is a critical security step to prevent unauthorized access
+
+2. **Deployment Options**:
+
+   **Option 1: Disable All Flows (Recommended)**
+   This approach starts with maximum security by disabling all authentication methods:
+   ```yaml
+   overrides:
+     keycloak:
+       keycloak:
+         values:
+           - path: realmInitEnv
+             value:
+               OPENTOFU_CLIENT_ENABLED: true
+           - path: realmAuthFlows
+             value:
+               USERNAME_PASSWORD_AUTH_ENABLED: false
+               X509_AUTH_ENABLED: false
+               SOCIAL_AUTH_ENABLED: false
+               OTP_ENABLED: false
+               WEBAUTHN_ENABLED: false
+               X509_MFA_ENABLED: false
+   ```
+   This is the most secure approach but requires OpenTofu to enable specific authentication methods after deployment.
+
+   **Option 2: Configure Final Flows Upfront**
+   If you know your exact authentication requirements, you can configure them directly:
+   ```yaml
+   overrides:
+     keycloak:
+       keycloak:
+         values:
+           - path: realmInitEnv
+             value:
+               OPENTOFU_CLIENT_ENABLED: true
+           - path: realmAuthFlows
+             value:
+               USERNAME_PASSWORD_AUTH_ENABLED: true
+               X509_AUTH_ENABLED: false
+               SOCIAL_AUTH_ENABLED: false
+               OTP_ENABLED: true
+               WEBAUTHN_ENABLED: false
+               X509_MFA_ENABLED: false
+   ```
+   This approach is simpler initially but may require manual steps if your requirements change.
+
+3. **Security Considerations**
+   - The `uds-opentofu-client` has elevated permissions - protect its credentials
+   - Never modify or delete the `uds-operator` clients as they are critical for system operation
+   - Monitor authentication logs after deployment for any unexpected access attempts
+
+4. **Verification**
+   - Verify all auth flows are as expected before deploying UDS Core
+   - Test authentication in a non-production environment first
+   - For detailed information on available authentication flows, see [Authentication Flow Documentation](./authentication-flows.md)
```

Comment:
```suggestion
   - For detailed information on available authentication flows, see [Authentication Flow Documentation](../authentication-flows)
```
Think this is the proper link ref in testing to make sure this works on the docs site.